src/Core/Util/DateTime/DateTime.php

```diff
@@ -33,10 +33,15 @@
  */
 final class DateTime
 {
+    /**
+     * Default format for date time string
+     */
+    public const DEFAULT_DATE_FORMAT = 'Y-m-d';
```

Comment:
Should'nt this depend on locale and CLDR ? 🤔

Reply:
No because it's the format used to store in database